com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs

```diff
@@ -557,11 +557,12 @@ void SortCameras(Camera[] cameras)
 
 #endif
 
-        static GraphicsFormat MakeRenderTextureGraphicsFormat(bool isHdrEnabled, bool needsAlpha)
+        internal static GraphicsFormat MakeRenderTextureGraphicsFormat(bool isHdrEnabled, HDRFormat requestHdrFormat, bool needsAlpha)
         {
             if (isHdrEnabled)
             {
-                if (!needsAlpha && RenderingUtils.SupportsGraphicsFormat(GraphicsFormat.B10G11R11_UFloatPack32, FormatUsage.Linear | FormatUsage.Render))
+                // TODO: we need a proper format scoring system. Score formats, sort, pick first or pick first supported (if not in score).
+                if (!needsAlpha && requestHdrFormat != HDRFormat.High && RenderingUtils.SupportsGraphicsFormat(GraphicsFormat.B10G11R11_UFloatPack32, FormatUsage.Linear | FormatUsage.Render))
```

Comment:
Feedback? so we could just cast the HDRFormat to GraphicsFormat here or similar sitautions, since it's values are GraphicsFormat. Opinions?

Also adding formats this way doesn't really scale, we need a better system if we want more formats cleanly.

Reply:
I think casting makes sense.